src/Illuminate/Support/Number.php

```diff
@@ -129,6 +130,10 @@ public static function percentage(int|float $number, int $precision = 0, ?int $m
             $formatter->setAttribute(NumberFormatter::FRACTION_DIGITS, $precision);
         }
 
+        if ($scaled) {
+            return $formatter->format($number);
+        }
+
         return $formatter->format($number / 100);
```

Comment:
We could do the following instead, in case, we want to do something with the formatted output or the like in the future:
```suggestion
        if (!$scaled) {
            $number /= 1000;
        }

        return $formatter->format($number);
```

Reply:
This was close to my original implementation but breaks backward compatibility. I do like this much better though. Should I refactor and point to the master branch instead of 11.x?